CONTRIBUTING.md

```diff
@@ -46,8 +46,24 @@ Finally, if you really, really, _really_ love developing PyBaMM, have a look at
 
 ## Installation
 
-To install PyBaMM with all developer options, type:
+To install PyBaMM with all developer options you can follow any one of the two steps described below:
```

Comment:
overall I'd keep instructions in the `contributing.md` to a minimum and put a link to the documentation that describes
the installation of PyBaMM in more details: https://pybamm.readthedocs.io/en/latest/install/install-from-source.html

Reply:
@tlestang thanks for the review! Should I completely remove the without tox installation?

Reply:
I think so. What are you thoughts on this, as a new PyBaMM developer? My idea is that it would be best to keep the installation on the `CONTRIBUTING.md` at a minimum, only describing the most convenient approach to set things up (which is to use tox) and putting a link to https://pybamm.readthedocs.io/en/latest/install/install-from-source.html for more details. It should be mentioned that tox must be installed separately though (as you did)

Reply:
Yes that is what I think now, as keeping the same installation process at 2 places would be repetitive. I will update the link quick.

Reply:
Ok great, thank you